scripts/configuration/ac-build-config.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <config>
     <sdkInfo>
-        <sdkVersion>1.13.1-SNAPSHOT</sdkVersion>
+        <sdkVersion>1.13.2-SNAPSHOT</sdkVersion>
         <iosVersion>1.13.0</iosVersion>
```

Comment:
You forgot to update the iOS version in a PR before.

Reply:
i have updated these changes